posts/ja/2023-01-31-22.0.0.12_jp.adoc

```diff
@@ -0,0 +1,277 @@
+---
+layout: post
+title: "CVE and many other notable bug fixes in Open Liberty 22.0.0.12"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: CVE and many other notable bug fixes in Open Liberty 22.0.0.12 - OpenLiberty.io
```

Comment:
`seo-title` needs to be translated from English to Japanese

Reply:
Hi @una-tapa and @takakiyo, I have used machine translation for the proposed Japanese translation. Can you help me check if the translation is correct? Thank you
```suggestion
seo-title: CVE およびその他の注目すべきバグ修正 (Open Liberty 22.0.0.12 - OpenLiberty.io)
```

Reply:
I just noticed @takakiyo provided the translation for this sentence in another spot in the blog post. I will use this translation instead of the machine translation.
```suggestion
seo-title: Open Liberty 22.0.0.12における脆弱性と主なバグの修正 - OpenLiberty.io
```